gallery/src/CodeExample.tsx

```diff
@@ -31,9 +31,14 @@ const CodeExample: FunctionComponent<Props> =
     const licenseStrippedContent = removeLicense(content);
 
     return (
-      <SyntaxHighlighter language={language || 'tsx'} style={atomDark}>
-        {licenseStrippedContent}
-      </SyntaxHighlighter>
+      <>
+        <h4 className="nx-h4">Code snippet:</h4>
+        <div className="gallery-code-box">
+          <SyntaxHighlighter language={language || 'tsx'} style={atomDark}>
+            {licenseStrippedContent}
+          </SyntaxHighlighter>
+        </div>
+      </>
```

Comment:
This change overlaps and conflicts with the change I'm making in RS-125.

Reply:
> This change overlaps and conflicts with the change I'm making in RSC-125.

https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/8e6f6c5edbd70bda9634e3aab1d435a853ae7e18